projects/bp-gallery/package.json

```diff
@@ -85,7 +85,7 @@
     "build:css": "npx tailwindcss -i ./src/shared/tailwind.css -o ./src/index.css --watch",
     "test": "env-cmd -f ./environments/.env.development react-scripts test --watchAll",
     "test:ci": "env-cmd -f ./environments/.env.development react-scripts test --verbose",
-    "cy:run": "cypress run",
+    "cy:run": "cypress run -c video=false,videoUploadOnPasses=false,screenshotOnRunFailure=false",
```

Comment:
I'm removing the line that disables screenshots, but keeping the line that disables the videos since they don't have any value and increase the test run duration by a minute or more